docs/Quickstart.mdx

```diff
@@ -0,0 +1,112 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Quick Installation
+
+This is the quick installation guide for Enarx. The goal is to help you get started using Enarx right away for development/debugging purposes on Windows, macOS, and Linux.
+
+:::note
+To build from source, please refer to the [complete installation guide](https://enarx.dev/docs/Install).
+:::
+
+:::note
+Full [TEE](https://enarx.dev/docs/Start/TEE) (Intel SGX/AMD SEV-SNP) and KVM support is available with the published statically compiled Linux binary.
+:::
+
+<Tabs>
```

Comment:
From looking at the rendered tabs I'm concerned that they're not extremely visible. Can we add a line above the tabs saying something like "Please select your operating system:"?